src/main/java/net/minecraftforge/common/ToolActions.java

```diff
@@ -93,4 +113,5 @@
     public static final Set<ToolAction> DEFAULT_SHOVEL_ACTIONS = Stream.of(SHOVEL_DIG, SHOVEL_FLATTEN).collect(Collectors.toCollection(Sets::newIdentityHashSet));
     public static final Set<ToolAction> DEFAULT_PICKAXE_ACTIONS = Stream.of(PICKAXE_DIG).collect(Collectors.toCollection(Sets::newIdentityHashSet));
     public static final Set<ToolAction> DEFAULT_SWORD_ACTIONS = Stream.of(SWORD_DIG, SWORD_SWEEP).collect(Collectors.toCollection(Sets::newIdentityHashSet));
+    public static final Set<ToolAction> DEFAULT_SHEARS_ACTIONS = Stream.of(SHEARS_DIG, SHEARS_HARVEST, SHEARS_CARVE, SHEARS_DISARM).collect(Collectors.toCollection(Sets::newIdentityHashSet));
```

Comment:
you know we should probaby make a helper:
```private static Set<ToolAction> of(ToolAction... actions) {
    return Stream.of(actions).collect(Collectors.toCollection(Sets:newIdentityHashSet));
  }```
  To make these inits less spammy.

Reply:
That is a good idea. :)